src/github.ts

```diff
@@ -222,6 +222,101 @@ export const updateComment = async (
   }
 }
 
+export const reactToComment = async (
+  ctx: Context,
+  octokit: Octokit,
+  ...args: Parameters<typeof octokit.reactions.createForIssueComment>
+): Promise<number | undefined> => {
+  const response = await octokit.reactions.createForIssueComment(...args)
+
+  if (response.status === 201) {
+    return response.data.id
+  } else {
+    ctx.logger.error(response, `reactToComment response unsuccessful`)
+  }
+}
+
+export const removeReactionToComment = async (
+  ctx: Context,
+  octokit: Octokit,
+  ...args: Parameters<typeof octokit.reactions.deleteForIssueComment>
+): Promise<void> => {
+  const response = await octokit.reactions.deleteForIssueComment(...args)
+
+  if (response.status !== 204) {
+    ctx.logger.error(response, `reactToComment response unsuccessful`)
+  }
+}
+
+export const cleanComments = async (
```

Comment:
You could remove most of the function body, if you would use `opstooling-integrations`, which has `makePaginateMethod` helper. Just saying.